misk-jooq/build.gradle.kts

```diff
@@ -21,6 +21,7 @@ dependencies {
   api(Dependencies.wispLogging)
 
   testImplementation(Dependencies.assertj)
+  testImplementation(Dependencies.wispTimeTesting)
```

Comment:
Changing the implementation to api in the MiskTimeTesting module should let you get rid of all these added explicit testImplementation changes in other modules.

Reply:
I don't think minimizing the diff is the right optimization here. I think the patch as written is better because it makes it explicit that the FakeClock is coming from wisp rather than implicitly getting it transitively through a deprecated wrapper.